spec/facility_spec.rb

```diff
@@ -23,4 +34,133 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
-end
+
+  describe '#register vehicle' do
+    it 'may offer vehicle registration' do
+      expect(@facility_1.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+      expect(@cruz.registration_date).to be nil
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq 0
+
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
```

Comment:
So `register_vehicle(@cruz)` is a command that should update the `registered_vehicles` method. So you wouldn't want it how you have it written on line 45. I would update your test to read like
```ruby
@facility_1.register_vehicle(@cruz)
expect(@facility_1.registered_vehicles).to eq([@cruz])
```
You would want to update all the tests to emulate this.